lib/core/hooks.js

```diff
@@ -1,5 +1,5 @@
 import { resolve, posix } from 'path'
-import deepMerge from 'deepmerge'
+import merge from 'lodash.merge'
```

Comment:
Sorry to go back and forth on this but for the size reasons we've recently switched to `klona` in `nuxt-i18n` (https://github.com/nuxt-community/i18n-module/pull/981) and I'd like to do the same here.

Reply:
In fact, I think that Klona is for cloning, not merging. We need to merge multiple objects, so I don't think I can replace `lodash.merge` with `klona`.